packages/ibm-products/src/components/SteppedAnimatedMedia/SteppedAnimatedMedia.tsx

```diff
@@ -84,25 +84,27 @@ export const SteppedAnimatedMedia = React.forwardRef(
     }, [filePaths]);
 
     useEffect(() => {
-      const prefersReducedMotion = window?.matchMedia
-        ? window.matchMedia('(prefers-reduced-motion: reduce)').matches
-        : true;
-      if (localRefValue) {
-        animRef.current?.destroy();
-        animRef.current = lottie.loadAnimation({
-          container: localRefValue,
-          renderer: 'svg',
-          animationData: jsonData[clamp(playStep, 0, jsonData.length - 1)],
-          loop: false,
-          autoplay: false,
-          rendererSettings: {
-            preserveAspectRatio: 'xMidYMid slice',
-          },
-        });
-        prefersReducedMotion
-          ? animRef.current?.goToAndStop(0)
-          : animRef.current?.goToAndPlay(0);
-      }
+      import('lottie-web').then((lottie) => {
+        const prefersReducedMotion = window?.matchMedia
+          ? window.matchMedia('(prefers-reduced-motion: reduce)').matches
+          : true;
+        if (localRefValue) {
```

Comment:
i think this should be moved out before the import. the lottie code isn't even run unless `localRefValue` is true, so as an extra level of optimization i would wrap the import with this if statement.

Reply:
Hi David, 
Thats make sense, however `localRefValue` will never be null with the implementation here.
```
 const backupRef = useRef<HTMLDivElement>(null);
    const localRef = ref ?? backupRef;
```